taf/updater/updater.py

```diff
@@ -139,8 +139,9 @@ def _reset_repository(repo, commits_data):
 @define
 class UpdateConfig:
     operation: OperationType = field(converter=OperationType)
-    url: str = field(
-        metadata={"docs": "URL of the remote authentication repository"}, default=None
+    remote_url: str = field(
```

Comment:
Why do we need to separate remote and clone urls? We want to attempt to clone the repo from all remote urls

Reply:
I believe it's related to `taf repo clone` only taking in a single URL, that is then passed down to all child dependent repositories as well. It appears that we have [this](https://github.com/openlawlibrary/taf/blob/692c469ec7328f5eee9eeef23e25366e28e1b42e/taf/updater/updater.py#L484) and it does not seem very flexible in case of dependent authentication repositories. 

Do you have a particular solution in mind? If not, I'd vote we sort out in a separate issue.

Reply:
So `clone` iterates over all urls and attempts to clone from each of them, until it succeeds in cloning the repo. The reason why there's only one auth repo url is that when you run `taf repo clone` you only specify one url. I never really put too much thought into cloning dependencies. I vote for replacing this existing url with a list of urls.